doc/chomp_interface_tutorial.rst

```diff
@@ -12,33 +12,40 @@ Once you have this package simply run::
 
-Using CHOMP with your own robot
--------------------------------
+Step to apply CHOMP
+====================
 
-1. Simply download `chomp_planning_pipeline.launch.xml <https://github.com/ros-planning/moveit_resources/blob/master/fanuc_moveit_config/launch/chomp_planning_pipeline.launch.xml>`_ file into the launch directory of your moveit config package. So into the *jaco_moveit_config/launch* directory.
-2. Adjust the line <rosparam command="load" file="$(find moveit_resources)/fanuc_moveit_config/config/chomp_planning.yaml" /> to <rosparam command="load" file="$(find jaco_moveit_config)/config/chomp_planning.yaml" />
-3. Download `chomp_planning.yaml <https://github.com/ros-planning/moveit_resources/blob/master/fanuc_moveit_config/config/chomp_planning.yaml>`_ file into the config directory of your moveit config package. So into the *jaco_moveit_config/config* directory.
-4. Copy the *demo.launch* file to *demo_chomp.launch*. Note that this file is also in the launch directory of the *jaco_moveit_config* package.
+1. Simply download `chomp_planning_pipeline.launch.xml <https://github.com/ros-planning/moveit_resources/blob/master/fanuc_moveit_config/launch/chomp_planning_pipeline.launch.xml>`_ file into the launch directory of your moveit config package. So into the *fetch_moveit_config/launch* directory.
+2. Adjust the line `<rosparam command="load" file="$(find moveit_resources)/fanuc_moveit_config/config/chomp_planning.yaml" />` to `<rosparam command="load" file="$(find fetch_moveit_config)/config/chomp_planning.yaml" />`
+3. Download `chomp_planning.yaml <https://github.com/ros-planning/moveit_resources/blob/master/fanuc_moveit_config/config/chomp_planning.yaml>`_ file into the config directory of your moveit config package. So into the *fetch_moveit_config/config* directory.
+4. Copy the *demo.launch* file to *demo_chomp.launch*. Note that this file is also in the launch directory of the *fetch_moveit_config* package.
 5. Find the lines where *move_group.launch* is included and change it to::
 
-    <include file="$(find jaco_moveit_config)/launch/move_group.launch">
+    <include file="$(find fetch_moveit_config)/launch/move_group.launch">
       <arg name="allow_trajectory_execution" value="true"/>
       <arg name="fake_execution" value="true"/>
       <arg name="info" value="true"/>
       <arg name="debug" value="$(arg debug)"/>
-      <arg name="planner" value="chomp" />
+      <arg name="pipeline" value="chomp" />
     </include>
 
-You probably only need to change the planner arg to chomp.
+You probably only need to change/add the `pipeline` arg to `chomp`.
 
 6. Run the demo::
        
-    roslaunch jaco_moveit_config demo_chomp.launch
+    roslaunch fetch_moveit_config demo_chomp.launch
+
+7. On `Context` tab on RViz, you should be able to choose `CHOMP` in `Planning Library` pulldown menu (see the image).
+
+.. image:: rviz_fetch_chomp_initialized.png
+   :width: 700px
```

Comment:
Shouldn't there be a step 8 that says something like "now run a plan using interactive markers to test"?